package.json

```diff
@@ -32,6 +32,7 @@
     "test-integration": "ava -v -s test/*/*.integration.test.js && ava -v test/e2e.integration.test.js",
     "test-unit": "ava -v",
     "docker-reset": "docker-compose down && docker rmi $(docker images -a -q) --force",
+    "docker-down": "npm run db-reset && docker-compose down",
```

Comment:
Are we sure this will actually reset the databases used in docker? Looks like `npm run db-reset` will call `npm run mongo-reset && npm run redis-flush`

Reply:
We don't have any volumes for mongo or redis so I _think_ `docker-compose down` should be sufficient to reset database

Reply:
sorry, don't we have `ledger-redis`?

Reply:
ah. gotcha